packages/components/src/components/Input/Input.tsx

```diff
@@ -0,0 +1,118 @@
+import React from "react";
+import { InputBox } from "../InputBox";
+import { Box } from "../../primitives/Box";
+
+export type InputTypes =
+  | "date"
+  | "email"
+  | "hidden"
+  | "number"
+  | "password"
+  | "search"
+  | "tel"
+  | "text"
+  | "time";
+
+export interface InputTypeProps {
+  type: InputTypes;
+  inputmode?: string | undefined;
```

Comment:
WHy do we need the undefined if it is already an optional prop?

Reply:
That's a miss on my part. I'll remove it.